### PR TITLE
Move Auth Methods section from Session Sigs to User Wallets

### PR DESCRIPTION
**Pull request by Aaryan-R-S**
### Fixes Issue: [LIT-2507](https://linear.app/litprotocol/issue/LIT-2507/moving-auth-methods-section-from-session-sigs-to-user-wallets-auth)

# Description

Moving Auth Methods section from Session Sigs to User Wallets->Auth Methods

This page in itself is already quite buried in the docs: https://developer.litprotocol.com/v3/sdk/authentication/session-sigs/auth-methods/overview/ (have to go through a lot of content to get here)

We should perhaps consider moving the 'auth methods' section (that covers social login, webauthn, and otp / sms) to within the PKPs section (perhaps as a subset of this section https://developer.litprotocol.com/v3/sdk/wallets/auth-methods)

Trying to update docs for it!

Fixes  # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Introducing new feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

File: docs/sdk/authentication/session-sigs/walletless-sigs.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Walletless Signatures
+
+With Lit Protocol, you can seamlessly onboard users into web3 using familiar authentication flows like social login and passkeys. The Lit SDK simplifies the process of generating and presenting signatures derived from authentication methods that don't require users to have an existing web3 wallet. Currently, Lit Protocol supports the following non-wallet authentication methods:
+
+- [Social Login (e.g., Google, Discord)](../../wallets/auth-methods/lit-auth-methods/social-login)
+- [WebAuthn](../../wallets/auth-methods/lit-auth-methods/web-authn)
+- [Email / SMS](../../wallets/auth-methods/lit-auth-methods/email-sms)
+
+Support for one-time passcodes (OTPs) through email and SMS is coming soon.
```
Comment (a1ttech) [inline docs/sdk/authentication/session-sigs/walletless-sigs.md:13]: no longer coming soon, it is live :)

File: docs/sdk/authentication/session-sigs/walletless-sigs.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+By utilizing PKP authentication methods, you can build frictionless and secure experiences in the decentralized web without the complexities of private key management.
```
Comment (a1ttech) [inline docs/sdk/authentication/session-sigs/walletless-sigs.md:15]: and without trusting a centralized custodian with your data ;)

File: docs/sdk/authentication/session-sigs/walletless-sigs.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+:::note
+
+Generating `SessionSigs` using various authentication methods is still in active development, so things may change. To stay up-to-date, always use the latest version of the Lit JS SDK.
```
Comment (a1ttech) [inline docs/sdk/authentication/session-sigs/walletless-sigs.md:19]: @Adarsh-Kumar28 do you think we should we remove the 'active development' tag?

File: docs/sdk/wallets/auth-methods/custom-auth.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ Inside of your Lit Actions, there is an object called `Lit.Auth` that will be pr
...
 - authSigAddress: A verified wallet address, if one was passed in. This is the address that was used to sign the AuthSig.
-- authMethodContexts: An array of auth method contexts. Each entry will contain the following items: `userId`, `appId`, and `authMethodType`. A list of AuthMethodTypes can be found [here](../../../../sdk/wallets/auth-methods) in the docs and is used [here](https://github.com/LIT-Protocol/LitNodeContracts/blob/main/contracts/PKPPermissions.sol#L25) in the PKPPermissions Contract.
+- authMethodContexts: An array of auth method contexts. Each entry will contain the following items: `userId`, `appId`, and `authMethodType`. A list of AuthMethodTypes can be found [here](../auth-methods) in the docs and is used [here](https://github.com/LIT-Protocol/LitNodeContracts/blob/main/contracts/PKPPermissions.sol#L25) in the PKPPermissions Contract.
```
Comment (a1ttech) [inline docs/sdk/wallets/auth-methods/custom-auth.md:31]: can we make sure we are linking the most up to date PKPPermissions contract?

File: docs/sdk/wallets/auth-methods.md [hunk 5]
```diff
@@ -9,7 +9,9 @@ Authentication methods are ways of asigning Programmable Key Pairs (PKP) to a sp
 
 An authentication method refers to the specific credential (i.e a wallet address, Google oAuth, or Discord account) that is programmatically tied to the PKP and used to control the underlying key-pair. Only the auth method associated with a particular PKP has the ability to combine the underlying shares. You can read more about how authentication works with PKPs on our [blog](https://spark.litprotocol.com/how-authentication-works-with-pkps/).
 
-Right now, there are two main ways to do auth with Lit Actions. We will dive into these two methods below.
+Right now, there are two main ways to do auth with Lit Actions. We will dive into these two methods below:
+1. [Using Lit Auth Directly](#using-lit-auth-directly)
+2. [Custom Auth / Adding new Auth Methods not yet supported by Lit](#custom-auth--adding-new-auth-methods-not-yet-supported-by-lit)
```
Comment (a1ttech) [inline docs/sdk/wallets/auth-methods.md:14]: here let's just link to the custom auth page directly, not another redirect

File: docs/sdk/wallets/auth-methods/overview.md [hunk 6]
```diff
@@ -6,9 +6,9 @@ sidebar_position: 1
 
 With Lit Protocol, you can seamlessly onboard users into web3 using familiar authentication flows like social login and passkeys. The Lit SDK simplifies the process of generating and presenting signatures derived from authentication methods that don't require users to have an existing web3 wallet. Currently, Lit Protocol supports the following non-wallet authentication methods:
```
Comment (a1ttech) [inline docs/sdk/wallets/auth-methods/overview.md:7]: i think this page as a whole is repetitive. I think the 'auth-methods.md' page serves as a good enough intro, i don't think we need to include this overview page as well

File: docs/sdk/authentication/session-sigs/walletless-sigs.md (hunk 1 above)
Comment (Aaryan-R-S) [reply docs/sdk/authentication/session-sigs/walletless-sigs.md:13]: Oh I see.
Done @a1ttech!

File: docs/sdk/authentication/session-sigs/walletless-sigs.md (hunk 2 above)
Comment (Aaryan-R-S) [reply docs/sdk/authentication/session-sigs/walletless-sigs.md:15]: Added @a1ttech :)

File: docs/sdk/authentication/session-sigs/walletless-sigs.md (hunk 3 above)
Comment (Aaryan-R-S) [reply docs/sdk/authentication/session-sigs/walletless-sigs.md:19]: Agree!
Done @a1ttech :)

File: docs/sdk/wallets/auth-methods/overview.md [hunk 7]
```diff
@@ -6,9 +6,9 @@ sidebar_position: 1
...
 
-- [Social Login (e.g., Google, Discord)](../auth-methods/social-login.md)
-- [WebAuthn](../auth-methods/web-authn.md)
-- [Email / SMS](../auth-methods/email-sms.md)
+- [Social Login (e.g., Google, Discord)](lit-auth-methods/social-login)
+- [WebAuthn](lit-auth-methods/web-authn)
+- [Email / SMS](lit-auth-methods/email-sms)
 
 Support for one-time passcodes (OTPs) through email and SMS is coming soon.
```
[the remark this answers is not in the document]
Comment (Aaryan-R-S) [reply docs/sdk/wallets/auth-methods/overview.md:13]: Done @a1ttech :)

File: docs/sdk/wallets/auth-methods/custom-auth.md (hunk 4 above)
Comment (Aaryan-R-S) [reply docs/sdk/wallets/auth-methods/custom-auth.md:31]: Done @a1ttech :)

File: docs/sdk/wallets/auth-methods.md (hunk 5 above)
Comment (Aaryan-R-S) [reply docs/sdk/wallets/auth-methods.md:14]: Done @a1ttech :)

File: docs/sdk/wallets/auth-methods/overview.md (hunk 6 above)
Comment (Aaryan-R-S) [reply docs/sdk/wallets/auth-methods/overview.md:7]: Completely agree!
Done @a1ttech :)

**Review by a1ttech — APPROVED** [review]

**Pull request merged**